build.gradle

```diff
@@ -1,5 +1,5 @@
 allprojects {
-    ext.kotlin_version = '1.5.21'
+    ext.kotlin_version = '1.9.10'
```

Comment:
have u checked ktlint new versiion? https://github.com/pinterest/ktlint/releases

Reply:
will create another issue and resolve the detekt and ktlint dependencies.